app/repositories/post.py

```
@@ -0,0 +1,272 @@
+            "views": 0,
+            "likes": 0,
+            "comments_count": 0,
+            "created_at": now,
+            "updated_at": now
+        }
+        
+        return self.create(post_data)
+    
+    # 수정 메서드
+    
+    def update_post(self, post_id: int, **updates) -> bool:
```

Ruff (0.14.11)

Missing type annotation for `**updates`

(ANN003)